app/controllers/crud_controller.rb

```diff
@@ -143,7 +143,11 @@ def index_path
 
   # Access params for model
   def permitted_params
-    params.require(model_identifier).permit(permitted_attrs)
+    params.require(model_identifier).permit(permitted_globalized_attrs)
```

Comment:
Ich frag mich noch ob es hier Sinn machen würde, beim bestehenden API zu bleiben und ggf. den Globalisierungs Aspekt im Controller in einen Concern zu extrahieren